packages/core/src/boot/init.ts

```diff
@@ -35,7 +35,6 @@ export function defineGlobal<Global, Name extends keyof Global>(global: Global,
 
 export enum BuildMode {
   RELEASE = 'release',
-  STAGING = 'staging',
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏it is still used in gitlab
https://github.com/DataDog/browser-sdk/blob/c13c21769bbc9ebcc5e2ebaa767dba1e6501537b/.gitlab-ci.yml#L237